blueprints/emberfire-torii/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  description: 'Generates a defautl torri adapter to authenticate with Firebase.',
```

Comment:
spelling: "default" and "Torii"

(I mistype torii **every single time**, too :crying_cat_face: )